src/content/fr/2020/performance.md

```diff
@@ -0,0 +1,388 @@
+
+### LCP par répartition géographique
+
+{{ figure_markup(
+  image="performance-lcp-by-geo.png",
+  caption="Les performances agrégées du LCP réparties par pays.",
+  description="Diagramme à barres montrant que les pourcentages les plus élevés de bonnes performances du LCP sont répartis entre les pays européens et asiatiques. La République de Corée est en tête avec 76 % de bonnes mesures, tandis que l’Inde est en dernière position avec 16 % de bonnes mesures.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=1936626175&format=interactive",
+  width="645",
+  height="792",
+  sheets_gid="263037691",
+  sql_file="web_vitals_by_country.sql"
+  )
+}}
+
+Le pourcentage le plus élevé de bonnes mesures du LCP est principalement réparti entre les pays européens et asiatiques, la République de Corée (Corée du Sud) étant en tête avec 76 % de bonnes mesures. La Corée du Sud est un leader constant en matière de vitesse de téléphonie mobile, avec une impressionnante vitesse de téléchargement de 145 Mbps <a href="https://www.speedtest.net/global-index" hreflang="en">selon le Speedtest Global Index</a> pour le mois d’octobre. Le Japon, la République tchèque, Taïwan, l’Allemagne et la Belgique sont également quelques pays qui offrent des débits mobiles élevés et fiables. L’Australie, bien que leader en matière de vitesse des réseaux mobiles, est désavantagée par la lenteur des connexions de bureau et la latence, ce qui la place dans la partie inférieure du classement ci-dessus.
+
+L’Inde reste la dernière de notre série de données, avec seulement 16 % de bonnes expériences. Alors que la population de personnes se connectant à Internet pour la première fois ne cesse de croître, les vitesses des réseaux mobiles et de bureau sont <a href="https://www.opensignal.com/reports/2020/04/india/mobile-network-experience" hreflang="en">toujours un problème</a>, avec des téléchargements moyens de 10 Mbps pour la 4G, 3 Mbps pour la 3G et moins de 50 Mbps pour le bureau.
+
+### LCP par type de connexion
+
+{{ figure_markup(
+  image="performance-lcp-by-connection-type.png",
+  caption="Performance agrégée des LCP répartie par type de connexion.",
+  description="Diagramme à barres montrant que 48 % des sites web ont un bon LCP sur la 4G. Le nombre de sites web bien notés diminue à 8 % sur la 3G, 1 % sur la 2G, 0 % sur la 2G lente et 18 % sur les connexions hors ligne.",
```

Comment:
Should be have `&nbsp;` before the %'s?

Note this applies throughout.

Reply:
They are here, you just can't see them in Github. If you're using VSCode, may I recommend the Gremlins extension?

Reply:
Ah I never thought to look for non-encoded versions of the non-breaking space. Grand - resolving!